p3/views/profile.py

```diff
@@ -161,3 +162,31 @@ def p3_account_spam_control(request):
         if form.is_valid():
             form.save()
     return render(request, "assopy/profile_spam_control.html", ctx)
```

Comment:
it's the django app that takes care of user accounts, invoices, payments, orders, etc...